hackIDE/static/hackIDE/js/custom.js

```diff
@@ -502,7 +517,14 @@ $(document).ready(function(){
 		downloadFile("code", editorContent, $("#lang").val());
 
 	});
+	
+	//when download-zip is clicked
+	$("#download-zip").click(function(){
```

Comment:
Instead of creating a new trigger please replace the old download with your own implementation. Let me know if this isn't clear. Keep pushing to your master branch and those commits will keep adding to this PR automatically.